app/dao/notifications_dao.py

```diff
@@ -353,11 +354,11 @@ def dao_timeout_notifications(timeout_period_in_seconds):
     updated = db.session.query(Notification). \
         filter(Notification.created_at < (datetime.utcnow() - timedelta(seconds=timeout_period_in_seconds))). \
         filter(Notification.status.in_([NOTIFICATION_CREATED, NOTIFICATION_SENDING, NOTIFICATION_PENDING])). \
```

Comment:
i think at the very least, we should be doing logging here - if somethings in `sending`/`pending` then the fault is with our provider. if something's in `created`, we've dropped the ball.